src/plugins/messenger/MessengerPreview/components/MessengerSubtitle.tsx

```diff
@@ -1,11 +1,13 @@
 import { MessagePluginFactoryProps } from "../../../../common/interfaces/message-plugin";
+import { IWebchatConfig } from "../../../../common/interfaces/webchat-config";
 
 export const getMessengerSubtitle = ({ React, styled }: MessagePluginFactoryProps) => {
-    const MessengerSubtitle = styled.p({
+    const MessengerSubtitle = styled.p<{ config: IWebchatConfig }>(props => ({
         margin: 0,
         color: 'hsla(0, 0%, 0%, .54)',
-        fontSize: 13
-    });
+        fontSize: 13,
+        whiteSpace: props.config.settings.ignoreLineBreaks ? "initial" : "pre-wrap" ,
```

Comment:
We should use "pre-line" instead of "pre-wrap". "pre-wrap" also changes the way sequence white spaces are treated which seems to be an unwanted change for this bugfix.

Reply:
fixed